isb_web/main.py

```diff
@@ -102,16 +66,9 @@
 STAC_COLLECTION_URL_PATH = config.Settings().stac_collection_url_path
 
 app = fastapi.FastAPI(openapi_tags=tags_metadata)
+manage_app = manage.manage_api
 dao = SQLModelDAO(None)
 
-app.add_middleware(
-    fastapi.middleware.cors.CORSMiddleware,
-    allow_origins=["*"],
-    allow_methods=["*"],
-    allow_headers=["*"],
-)
-app.add_middleware(SessionMiddleware, secret_key=COOKIE_SECRET)
-
```

Comment:
For future reference, the middleware of the main app *does* affect any registered sub apps!